### PR TITLE
MINOR: Follow-up Update on 1.0.0 release

### PR DESCRIPTION
**Pull request by guozhangwang**
1. Replaced 100 folder with 10 folder.
2. Update Streams/index.html file with a couple follow-up PRs that went in post RC4.
3. Add the paragraph that Sally and Jun suggested in release notes.

@ijuma @rajinisivaram

### CODE REVIEW

File: 10/documentation.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+<!--#include virtual="../includes/_top.htm" -->
+
+
+<div class="content documentation documentation--current">
+	<!--#include virtual="../includes/_nav.htm" -->
+	<div class="right">
+		<!--#include virtual="../includes/_docs_banner.htm" -->
+    <h1>Documentation</h1>
+    <h3>Kafka 0.11.0 Documentation</h3>
+    Prior releases: <a href="/07/documentation.html">0.7.x</a>, <a href="/08/documentation.html">0.8.0</a>, <a href="/081/documentation.html">0.8.1.X</a>, <a href="/082/documentation.html">0.8.2.X</a>, <a href="/090/documentation.html">0.9.0.X</a>, <a href="/0100/documentation.html">0.10.0.X</a>, <a href="/0101/documentation.html">0.10.1.X</a>, <a href="/0102/documentation.html">0.10.2.X</a>.
```
Comment (ijuma) [inline 10/documentation.html]: This needs to be updated.

File: downloads.html [hunk 2]
```diff
@@ -9,48 +9,75 @@
...
-							</ul>
-							We build for multiple versions of Scala. This only matters if you are using Scala and you want a version built for the same Scala version you use. Otherwise any version should work (2.11 is recommended).
-					</li>
-			</ul>
+        <ul>
+            <li>
+                Released November 1, 2017
+            </li>
+            <li>
+                Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/1.0.0/kafka-1.0.0-src.tgz">kafka-1.0.0-src.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/1.0.0/kafka-1.0.0-src.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/1.0.0/kafka-1.0.0-src.tgz.md5">md5</a>)
```
Comment (ijuma) [inline downloads.html]: I was linking to the sha512 file since md5 is deprecated these days.

File: downloads.html [hunk 3]
```diff
@@ -9,48 +9,75 @@
...
+        <h3>0.11.0.1</h3>
+        <ul>
+            <li>
+                Released September 13, 2017
+            </li>
+            <li>
+                <a href="https://archive.apache.org/dist/kafka/0.11.0.1/RELEASE_NOTES.html">Release Notes</a>
+            </li>
+            <li>
+                Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/0.11.0.1/kafka-0.11.0.1-src.tgz">kafka-0.11.0.1-src.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/0.11.0.1/kafka-0.11.0.1-src.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/0.11.0.0/kafka-0.11.0.0-src.tgz.md5">md5</a>)
```
Comment (ijuma) [inline downloads.html]: There's a bug where 0.11.0.0 is used in one case.

**Review by ijuma — COMMENTED** [review]
Thanks for the PR. A few things I noticed.

File: 10/documentation.html (hunk 1 above)
Comment (guozhangwang) [reply 10/documentation.html]: ack.

File: downloads.html (hunk 2 above)
Comment (guozhangwang) [reply downloads.html]: ack.

**Comment by guozhangwang** [comment]
@ijuma addressed your comments.

Will submit another PR for kafka repo for the same necessary changes.

File: 10/documentation.html [hunk 4]
```diff
@@ -69,15 +69,15 @@
...
+    <h2><a id="streams" href="/0110/documentation/streams">9. Kafka Streams</a></h2>
     <p>
         Kafka Streams is a client library for processing and analyzing data stored in Kafka. It builds upon important stream processing concepts such as properly distinguishing between event time and processing time, windowing support, exactly-once processing semantics and simple yet efficient management of application state.
     </p>
     <p>
         Kafka Streams has a <b>low barrier to entry</b>: You can quickly write and run a small-scale proof-of-concept on a single machine; and you only need to run additional instances of your application on multiple machines to scale up to high-volume production workloads. Kafka Streams transparently handles the load balancing of multiple instances of the same application by leveraging Kafka's parallelism model.
     </p>
 
-    <p>Learn More about Kafka Streams read <a href="/100/documentation/streams">this</a> Section.</p>
+    <p>Learn More about Kafka Streams read <a href="/0110/documentation/streams">this</a> Section.</p>
```
Comment (ijuma) [inline 10/documentation.html]: Seems like this is not right?

Comment (guozhangwang) [reply 10/documentation.html]: Just updated that part while looking on diff files :)

**Review by ijuma — APPROVED** [review]
LGTM, thanks.

File: 100/javadoc/script.js [hunk 5]
```diff
@@ -1,30 +0,0 @@
...
-        if (value == type) {
-            sNode.className = activeTableTab;
-            spanNode.innerHTML = tabs[value][1];
-        }
-        else {
-            sNode.className = tableTab;
-            spanNode.innerHTML = "<a href=\"javascript:show("+ value + ");\">" + tabs[value][1] + "</a>";
-        }
-    }
-}
```
Comment (ijuma) [inline 100/javadoc/script.js]: Hmm, is it OK that this file was deleted?

**Review by ijuma — APPROVED** [review]

**Comment by guozhangwang** [comment]
Merged to asf-site

**Pull request closed without merging**